internal/server/kong/ws/config/compat/compatibility_table.go

```diff
@@ -830,6 +831,36 @@ var (
 				},
 			},
 		},
+		{
+			Metadata: config.ChangeMetadata{
+				ID:       config.ChangeID("P135"),
+				Severity: config.ChangeSeverityError,
+				Description: "For the 'zipkin' plugin, " +
+					"'config.http_response_header_for_traceid' field is used " +
+					"in Kong Gateway versions >= 3.1. " +
+					"The plugin configuration has been updated to use " +
+					"'config.http_response_header_for_traceid' in the data-plane.",
+				Resolution: "Please update the configuration to use " +
```

Comment:
I'm confused. Shouldn't we make sure that a new field is removed from payloads destined to  older DPs? This rule is making sure that `http_response_header_for_traceid` is set to `X-B3-TraceId` for newer DPs, but I think this should actually make sure to remove it for pre-3.1 DPs, basically a copy&paste of [this rule](https://github.com/Kong/koko/blob/main/internal/server/kong/ws/config/compat/compatibility_table.go#L700-L725) for the specific new config field.

Reply:
<img width="240" alt="image" src="https://user-images.githubusercontent.com/5423475/199548496-3e593d8e-9dcd-42bf-b312-f22645234983.png">
I originally had a removal check (7f84e48) but I confused myself and removed it. I might add it back in after I go through [Alan's comment](https://github.com/Kong/koko/pull/496#discussion_r1010860736).